packages/client/src/client.js

```diff
@@ -617,6 +622,15 @@ export class PercyClient {
     });
   }
 
+  async getErrorAnalysis(errors) {
```

Comment:
This can also be shift to utils right?

Reply:
No, why?
It's used to hit Percy-api. It should be a client only.